bin/oneoff/back_populate_user_proficiencies_take_three

```diff
@@ -243,17 +250,20 @@ def rollup_monthly_snapshots
 end
 
 def update_database
-  user_proficiency_hash = {}
+  user_proficiency_data = []
   puts "READING DATA FROM #{AGGREGATE_FILENAME}..."
-
-  puts "READ DATA FROM #{AGGREGATE_FILENAME}."
+  CSV.foreach(AGGREGATE_FILENAME) do |row|
+    mapped_row = row[0..50].map(&:to_i) << row[51] << row[52]
+    user_proficiency_data << mapped_row
+  end
+  puts "READ DATA (#{user_proficiency_data.size} rows) FROM #{AGGREGATE_FILENAME}."
 
   slice_count = 0
   puts "UPDATING DB..."
   # Update the DB with the data in user_proficiency_hash.
   begin
-    user_proficiency_hash.values.each_slice(1_000) do |slice|
-      puts "  SLICE_COUNT: #{slice_count}..."
+    user_proficiency_data.each_slice(SLICE_SIZE) do |slice|
+      puts "  SLICE_COUNT: #{slice_count} of #{user_proficiency_data.length / SLICE_SIZE}..."
```

Comment:
This may round down rather than up, but I don't care, the logging is to get an idea for percentage completed.